Major Mapping Update to Arena

ArcanissKenku opened the pull request:
<!--
This is a semi-strict format, you can add/remove sections as needed but the order/format should be kept the same
Remove these comments before submitting
-->

# Description

This is a major update to the map Arena and my first pull request for mapping changes as well.

- Overhauled Kitchen and Botany
- Added Justice Department
- Added AI Sattelite below Mystagogue
- Added Gym/Sauna
- Added Ice Cream Shop
- Added Holopads to all departments and across station
- Room Tweaks to CMO and Mystagogue, Robotics and Dorms
- Medical near Evac now abandoned for renovation project - Paramed still has station within medical (unchanged)
- Moved Southern Solars 

- Added Black and Red Bowtie Clothing
- Added Soda Jerk Hat Clothing
- Added Ice Cream Sign

---

<!--
This is default collapsed, readers click to expand it and see all your media
The PR media section can get very large at times, so this is a good way to keep it clean
The title is written using HTML tags
The title must be within the <summary> tags or you won't see it
-->

<details><summary><h1>Media</h1></summary>
<p>

Updated Renderer
<img width="6752" height="5728" alt="Arena" src="https://github.com/user-attachments/assets/904dee88-2f7a-421a-8f97-6614ab20b780" />

Overhaul of Botany, Kitchen and added Ice Cream Shop
<img width="978" height="925" alt="canteen" src="https://github.com/user-attachments/assets/3be5a972-0070-43b6-ae87-a7ab1a98b734" />

Implementation of Justice Department rooms
<img width="676" height="579" alt="justice" src="https://github.com/user-attachments/assets/44058d41-f715-4b7f-b5fd-40bccebb628a" />

Addition of AI Satellite and adjustments to Mystagogue's Room
<img width="566" height="887" alt="AI" src="https://github.com/user-attachments/assets/65c2f345-0fe5-4b74-9267-a8f4ccda973d" />
 
Addition of Gym and Sauna
<img width="675" height="268" alt="gym-sauna" src="https://github.com/user-attachments/assets/da493185-ed00-420b-9d98-85b1a471cd4c" />

Changes to dorms
<img width="485" height="607" alt="dorms" src="https://github.com/user-attachments/assets/0ef4b5cc-77c2-4ecb-b312-c5b1da70e9ec" />

Addition of Black/Red Bowtie and Soda Jerk Hat + Ice Cream Sign
<img width="850" height="861" alt="image" src="https://github.com/user-attachments/assets/dfe283f2-6084-455d-ab13-068f79c965c5" />



</p>
</details>

---

# Changelog

<!--
You can add an author after the `:cl:` to change the name that appears in the changelog (ex: `:cl: Death`)
Leaving it blank will default to your GitHub display name
This includes all available types for the changelog
-->

:cl:
- tweak: Major Update to Arena Map
- add: Three new clothing items: Black/Red Boetie and Soda Jerk Hat. Currently found only within Arena station.

ArcanissKenku committed:
Added Ice Cream Sign

Added an Ice Cream Parlour Sign in a similar style to other department signs.

ArcanissKenku committed:
Merge branch 'Floof-Station:master' into Mapping

ArcanissKenku committed:
Added Soda Jerk Hat and Boeties

Added Soda Jerk Hat and Red/Black Boetie

These go along with the Ice Cream Shop sign to fit with the style.

ArcanissKenku committed:
Updated Arena

- Overhauled Kitchen and Botany
- Added Justice Department
- Added AI Satellite below Mystagogue
- Added Gym/Sauna
- Added Ice Cream Shop
- Added Holopads across station
- Room Tweaks to CMO and Mystagogue
- Moved Southern Solars

ArcanissKenku committed:
Merge branch 'Floof-Station:master' into Mapping

ArcanissKenku committed:
Update Arena.yml

Fixing some invalid issues.

Dunrab commented:
@ArcanissKenku Will look at this more in the morning but at quick glance I noticed you replaced the camera routers with the CJs room, where is the new camera router room?

ArcanissKenku commented:
@Dunrab New Camera Router room is in the AI Satellite - Decided to move the cameras there but keep the comms in Engineering like previously.

ArcanissKenku commented:
I've added all the media and screenshots to the Media Dropdown and it has a picture of AI there now

M3739 reviewed (changes requested):
Please provide proper attribution in the JSON files' `copyright` field.

ArcanissKenku committed:
Proper attribution added

Added proper attribution in Copyright to .json files

ArcanissKenku committed:
Merge branch 'Mapping' of https://github.com/ArcanissKenku/Floof-Station-Arcaniss into Mapping

ArcanissKenku committed:
Soda Jerk Hat

Correction of Attribution

M3739 reviewed (approved):
Attribution looks proper now!

Sprites are approved. Still will need Dunrab to give their review on the map.

Dunrab reviewed (approved):
Changes look good, I like the new service area with the ice cream bar and thanks for saying where you moved the camera routers to my tired brain missed it last night.

ArcanissKenku commented:
@Dunrab Lemme know if you're free in voice as well as I'd be happy to playtest a little more of this and see if there's anything missed alongside some better eyes.

ArcanissKenku commented:
Just working on one more change, found an issue with the crew monitoring server.

Dunrab commented:
> Just working on one more change, found an issue with the crew monitoring server.

Take your time.

ArcanissKenku committed:
Hotfix #1

- Fixes Crew Monitoring Server on Round Start
- Some Beacons named incorrectly

Still issue with specifically 'Camera Monitor Computers' not working on round restart, quick fix is to unwrench and re-wrench.

No issues with camera servers or cameras however, AI also still functions.

ArcanissKenku commented:
Hotfixed the Crew Monitoring Server now working at round start.

Spoke to both Fish and Dunrab and Camera Monitors not working from round start appears to be a common issue unsolved.

Simple fix of unwrenching and wrenching the computer however solves this at least. So happy to pull as is for now then.

southbridge-fur reviewed (changes requested):
Move the clothing items to a separate PR

ArcanissKenku commented:
@southbridge-fur They're implemented as part of the map to go with the Ice Cream Shop though, I'd like to keep them together because they're on theme and given a reason to be part of this map specifically. I'd rather keep it as part of the same PR because it makes more sense in this context why - especially with the sign added as well.

ArcanissKenku committed:
Merge branch 'Floof-Station:master' into Mapping

M3739 commented:
Heisentest. Error is unrelated to the PR.

ArcanissKenku committed:
Merge branch 'Floof-Station:master' into Mapping

M3739 commented:
Tests are green, rewrote the CL for clarity, since naturally, they would ask where it is in the loadout system.
